### PR TITLE
OSDOCS-10425:updates limitation of rollback to SDN from OVN-K

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-10425

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:
@anuragthehatter and @npinaeva PTAL

### CODE REVIEW

**Comment by anuragthehatter** [comment]
/lgtm

Thanks Joe

File: modules/nw-network-plugin-migration-process.adoc [hunk 1]
```diff
@@ -51,6 +51,11 @@ Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the {sdn}
 ifeval::["{context}" == "migrate-from-openshift-sdn"]
 If a rollback to OpenShift SDN is required, the following table describes the process.
 
+[IMPORTANT]
+====
+You must wait till the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
```
Comment (bhardesty) [inline modules/nw-network-plugin-migration-process.adoc:56]:
```suggestion
You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
```

**Review by bhardesty — COMMENTED** [review]
Looks good! Just one tiny wording suggestion.

File: modules/nw-network-plugin-migration-process.adoc [hunk 2]
```diff
@@ -51,6 +51,11 @@ Cluster:: As nodes reboot, the cluster assigns IP addresses to pods on the {sdn}
 ifeval::["{context}" == "migrate-from-openshift-sdn"]
 If a rollback to OpenShift SDN is required, the following table describes the process.
 
+[IMPORTANT]
+====
+You must wait until the migration process from OpenShift SDN to OVN-Kubernetes network plugin is successful before initiating a rollback.
+====
```
Comment (pliurh) [inline modules/nw-network-plugin-migration-process.adoc:57]: Ensure the network and machine-config cluster operators are not in the `progressing` or `degraded` state before starting rollback.

Comment (zshi-redhat) [reply modules/nw-network-plugin-migration-process.adoc:57]: @pliurh  I assume the a "successful" migraiton shall indicate that "the network and machine-config cluster operators are not in progressing or degraded state", wdyt?

**Pull request merged**